packages/mux-elements-codemod/src/index.ts

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
+
+// @ts-ignore
+// importing it as `process` doesn't preserve the argv properly
+import { argv } from 'node:process';
```

Comment:
non-blocking: Either here or in our build, should we make these a cjs module for greater compat across node versions folks might be using?